packages/@lwc/engine-server/src/__tests__/fixtures/light-dom-scoped-styles-render/modules/x/basic/basic.js

```diff
@@ -0,0 +1,10 @@
+import { LightningElement } from 'lwc';
+import template from './template.html';
+
+export default class Basic extends LightningElement {
+  static renderMode = 'light';
+
+  render() {
+    return template;
```

Comment:
Asserting that we understand that scoped styles are associated with this template which is returned dynamically from the `render` function.